systems/plants/test/urdf_manipulator_dynamics_test.cpp

```diff
@@ -0,0 +1,63 @@
+
```

Comment:
will clean that one up in my next pull.  
On Feb 16, 2015, at 11:22 AM, Twan Koolen <notifications@github.com> wrote:

> In systems/plants/test/urdf_manipulator_dynamics_test.cpp:
> 
> > @@ -0,0 +1,63 @@
> > +
> Filename should really be camelcase; I also see that there are a few other files that use underscores in systems/plants/test.
> 
> —
> Reply to this email directly or view it on GitHub.
>